galore/cli/galore.py

```diff
@@ -305,6 +307,27 @@ def get_parser():
         '--overlay_label', type=str, default=None,
         help='Legend label for data overlay'
         )
+    parser.add_argument(
+        "--no-total",
+        action="store_false",
+        dest="total",
+        help='If using --pdos, do not plot the total spectrum. '
+             '(Total spectrum will still be calculated to determine legend cutoff.)'
+        )
+    parser.add_argument(
+        '--legend-cutoff', type=float, default=0.02, dest="legend_cutoff",
+        help='Cutoff value as fraction of max y-value in the plot range that determines '
+             'if a line is given a legend label (default: 0.02)'
+        )
+    parser.add_argument(
+        "--no-fill",
+        action="store_false",
+        dest="fill",
+        help="Don't colour-fill the photoelectron spectrum plot",
```

Comment:
This is a PDOS-only feature, right? This should say something like "Don't colour-fill orbital projections in spectrum plot."

(Not all Galore plots are PES, if you don't set weighting you get a regular DOS/PDOS!)